specs/validator/tableland-openapi-spec.yaml

```diff
@@ -198,43 +227,38 @@ components:
             }
         schema:
           $ref: "#/components/schemas/Schema"
-    TxnReceipt:
+    TransactionReceipt:
       type: object
       properties:
-        tableId:
+        table_id:
```

Comment:
So I noticed in the parser, we call this component the tokenId (token_id), but in other places, we call this the tableId (table_id). We should make a final call on this and pick one or the other. cc @sanderpick and @brunocalza on that topic. I don't think I have a preference, except that perhaps tokenId is more "accurate" since the combination of chainId and tokenId is effectively the tableId?

Reply:
IMHO, I'd use `tableId` for the numeric identifier of the table in the underlying chain.
Using `tokenId` feels to me like leaking terminology from smart contracts to our protocol.

The usual way we explained the fully qualified name was `{prefix}_{chainId}_{tableId}`.  I usually prefer to say `tableName` or `fully qualified name` to refer to the chain-independent identifier of the table (e.g: `foo_1_100`).

Also, changing to `tokenId` is quite painful in the validator, since to leave everything coherent we need to do some reasonable amount of renaming and potential table migrations to leave columns with the right names. Not a big deal, but definitely we want to be sure if we want to change such core terminologies.

This is subjective, so just leaving my opinion.

Reply:
You know now that I think about it, I think we already had this conversation and tableId was the decision!